lib/src/Pos/Util/UnitsOfMeasure.hs

```diff
@@ -22,6 +38,79 @@ data UnitOfMeasure =
     | Bytes
     | Lovelace
     | LovelacePerByte
-    deriving (Show, Eq)
+    deriving (Show, Eq, Ord)
+
+instance Buildable UnitOfMeasure where
+    build = \case
+        Bytes           -> "bytes"
+        LovelacePerByte -> "lovelace/byte"
+        Lovelace        -> "lovelace"
```

Comment:
Indeed, just to be more consistent with the rest. I also hesitated with `lovelaces` though, I guess it makes sense not to pluralize it since it's a currency :thinking: ...